data/fajnzylber2020sars/fajnzylber2020sars.yaml

```diff
@@ -0,0 +1,1733 @@
+# yaml-language-server: $schema=../.schema.yaml
+title: SARS-CoV-2 viral load is associated with increased disease severity and mortality
+doi: 10.1038/s41467-020-19057-5
+description: >
+  The paper quantified SARS-CoV-2 viral load from participants with a diverse range
+  of COVID-19 disease severity, including those requiring hospitalization, outpatients
+  with mild disease, and individuals with resolved infections. Nasopharyngeal swabs,
+  oropharyngeal swabs, sputum, and urine were collected from hospitalized participants.
```

Comment:
We need to mention the collection of blood samples and the plasma samples are from blood samples.

Reply:
Good point!